src/watertap_contrib/reflo/solar_models/surrogate/flat_plate/data/pyssc.py

```diff
@@ -0,0 +1,534 @@
+import sys, os
```

Comment:
Is this directory the best place to have this file? Shouldn't it be at a higher level that can later be utilized for flat plate and other units?

Reply:
Yes next step I think is to be able to generate data using REFLO and the SolarBaseClass... This would be moved to a core directory at that point is my initial though.